packages/zipkin/index.d.ts

```diff
@@ -80,7 +80,7 @@ declare namespace zipkin {
 
       map<V>(fn: (value: T) => V): IOption<V>;
       ifPresent<V>(fn: (value: T) => V): IOption<V>;
```

Comment:
I just noticed that [None's implementation of ifPresent always returns undefined](https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin/src/option.js#L8) rather than IOption.  Would you like to deal with that in this PR?  If so, let me know your thoughts on how to proceed.  Thanks!

Reply:
Looking at usages of ifPresent in the zipkin-js codebase, it seems like the return value is always ignored.  That makes it seem like we could change the return type to `void`.  Thoughts?

Reply:
`ifPresent` was never intended to return a value. It's purely a way of performing side effects. The correct signature is indeed `void`.

Reply:
Update: Yup, there is actually a bug in the implementation. `Some` returns a value for `ifPresent`, but it shouldn't. I guess the type signature matches the implementation, so the bug is both in the implementation and the signature.